dynawo/sources/Common/test/Test.cpp

```diff
@@ -334,7 +334,11 @@ TEST(CommonTest, testSparseMatrix) {
   SparseMatrix smj;
   smj.init(3, 3);
   smj.changeCol();
+#ifndef _MSC_VER
   smj.addTerm(1, 1./0.);
+#else
+  smj.addTerm(1, 1.);
```

Comment:
We are testing the presence of a Nan in the matrix so the test is not equivalent

Reply:
I know but MSVC raise an error when it catches / 0. and I don't know how to ignore it...

Reply:
we can try to use NAN from math.h

Reply:
It was a test for inf and not nan. I just defined a variable with zero and the compiler does not complain this time